src/components/Header/Navigations/Navigations.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState,useCallback } from "react";
+import { useSelector } from "react-redux";
+import { withRouter } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import { LINKS_WITHOUT_AUTH, LINKS_WITH_AUTH } from "../../../const/routes";
```

Comment:
avoid relative path add like
if the location is in  
src/components>Header>Navigations/Navigations.js
use like
`import Navigations from components/Header/Navigations/Navigations;`

path from the src folder